### ISSUE TITLE
Sync electronics content catalog to electronics-spa content catalog

### ISSUE THREAD

**Issue by WeizhengSap** [SAP/cloud-commerce-spartacus-storefront#893]
At the beginning, electronics storefront and SPA use the same content catalog (electronics content catalog). We have spacceleratorsampleaddon, which updates electronics content catalog data to make SPA has some different components. (e.g. different SiteLogo).
![screen shot 2019-01-15 at 8 26 14 am](https://user-images.githubusercontent.com/44440575/51184041-27520a00-18a1-11e9-8731-4a94a5ba2578.png)

Each time, we modify the cms data, both sites (acc storefront and SPA) will be affected.

Then, we created a new site for SPA, called "electronics-spa", and new catalogs for SPA as well. The new SPA catalog is a sub-catalog of electronics content catalog. (Content in a parent catalog is also available to child content catalogs.). So, in the spacceleratorsampleaddon, we can replace the homepage, and some components in headers.
But, there are some limitations. 
1. In smartedit, we cannot edit components which are not owned by SPA. 
![screen shot 2019-01-15 at 8 51 53 am](https://user-images.githubusercontent.com/44440575/51184638-e529c800-18a2-11e9-80f4-10edbe0704fd.png)
2. In smartedit "PAGES", we only can see the SPA specific pages. To see other pages, we have to replace all of them.
![screen shot 2019-01-15 at 8 52 16 am](https://user-images.githubusercontent.com/44440575/51184693-11dddf80-18a3-11e9-98e2-790e8e20aea9.png)
3.  If we add a new content slot in template (add "StoreFinderSlot" in the SPA catalog), there are errors shown in smartedit. To fix this issue, we have to move "StoreFinderSlot" to the parent catalog.
![screen shot 2019-01-09 at 4 04 21 pm](https://user-images.githubusercontent.com/44440575/51184739-320d9e80-18a3-11e9-83d4-352ef94d8c83.png)

Because of these limitations, we will sync electronics content catalog to electronics-spa content catalog, then we can update the synced data in the SPA content catalog.
![screen shot 2019-01-15 at 8 26 24 am](https://user-images.githubusercontent.com/44440575/51184883-97fa2600-18a3-11e9-83d0-68f75fc61bd5.png)

**Comment by WeizhengSap** [comment]
PR is here: https://stash.hybris.com/projects/C3PO/repos/spacceleratorsamplesaddon/pull-requests/10/overview

To code reviewer: please pull my codes, and help me to format `SpaSampleAddOnSampleDataImportService`. My eclipse has some problems

**Comment by Xymmer** [comment]
once merged, can you update dev-com-2 server so we can try it out?

**Comment by hackergil** [comment]
After code reviewing, I did the following:
- Recreate a new hybris environment with the _spartacus_ recipe (including db initialization)
- Installed a fresh version of spa pointing to my local smartedit. A couple of steps changed to integrate with smartedit so I updated the docs accordingly)

* I was not able to edit components from smartedit
* I was not able to browse multiple pages of spa from smartedit

My guess is that the cronjob that syncs the catalogs didn't run. Maybe there's a step missing?
